ts/screens/authentication/LandingScreen.tsx

```diff
@@ -89,15 +98,28 @@ const LandingScreen: React.SFC<Props> = props => {
       </Content>
 
       <View footer={true}>
+        {isCIEAvailable && (
+          <Button
+            block={true}
+            primary={true}
+            iconLeft={true}
+            onPress={undefined} // here navigate to identity card check
+            testID="landing-button-login-cie"
+          >
+            <IconFont name="io-cie" color={variables.colorWhite} />
+            <Text>{I18n.t("authentication.landing.loginCie")}</Text>
+          </Button>
+        )}
+        <View spacer={true} />
         <Button
           block={true}
           primary={true}
           iconLeft={true}
           onPress={navigateToIdpSelection}
-          testID="landing-button-login"
+          testID="landing-button-login-spid"
         >
           <IconFont name="io-profilo" color={variables.colorWhite} />
```

Comment:
I know it's not code introduced with this pr, but can you update it?
```suggestion
          <IconFont name={"io-profilo"} color={variables.colorWhite} />
```